addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.scss

```diff
@@ -0,0 +1,32 @@
+.subtask_list {
+    margin-top: 6px;
+
+    .subtask_list_row {
+        display: grid; 
+        grid-template-columns: 20px auto auto;
+        column-gap: 4px;
+        padding: 4px 0px;
+
+        .subtask_state_widget_col {
+            display: flex;
+            align-self: center;
+            justify-self: center;
+        }
+
+        .subtask_user_widget_col {
+            align-self: center;
+            justify-self: end;
+        }
```

Comment:
Can be removed since you added `justify-self-end` and `align-self-center` in the div. 🙂